yb-voyager/src/adaptiveparallelism/adaptive_parallelism_test.go

```diff
@@ -30,17 +30,44 @@ type dummyTargetYugabyteDB struct {
 	cpuUsageSys1  float64
 	cpuUsageUser2 float64
 	cpuUsageSys2  float64
+
+	memAvailable1              int
+	memTotal1                  int
+	tserverRootMemConsumption1 int
+	tserverRootMemSoftLimit1   int
+
+	memAvailable2              int
+	memTotal2                  int
+	tserverRootMemConsumption2 int
+	tserverRootMemSoftLimit2   int
```

Comment:
Q. Is all this info also available in the dbside function you have implemented?

Reply:
Yes, https://github.com/yugabyte/yugabyte-db/commit/872b59e5371790141729bce864f6cf9374bc77f9 . 
Also added a comment in code with this sample output